tests/i18n/tests.py

```diff
@@ -1424,8 +1424,11 @@ def test_check_for_language(self):
         self.assertTrue(check_for_language('en'))
         self.assertTrue(check_for_language('en-us'))
         self.assertTrue(check_for_language('en-US'))
+        self.assertTrue(check_for_language('be'))
+        self.assertTrue(check_for_language('be@latin'))
```

Comment:
Added, but specifying charset is currently not supported (and IMHO does not make sense as Django enforces UTF-8 on locales).